EasyPost.nuspec

```diff
@@ -39,5 +41,7 @@
         <file src="lib\net\net5.0\EasyPost.XML" target="lib\net5.0" />
         <file src="lib\net\net6.0\EasyPost.dll" target="lib\net6.0" />
         <file src="lib\net\net6.0\EasyPost.XML" target="lib\net6.0" />
+        <file src="README.md" target="docs\" />
+        <file src="icon.png" target="images\" />
```

Comment:
These paths don't match up?

Reply:
These are internal paths inside the final NuGet package.